testing_and_setup/compass/ocean/jigsaw_to_MPAS/mesh_definition_tools.py

```diff
@@ -65,7 +65,16 @@ def mergeCellWidthVsLat(
     return cellWidthOut
 
 
-def EC_CellWidthVsLat(lat):
+def EC_CellWidthVsLat(lat,
+    cellWidthEq = 30.0,
+    cellWidthMidLat = 60.0,
+    cellWidthPole = 35.0,
+    latPosEq = 15.0,
+    latPosPole = 73.0,
+    latTransition = 40,
+    latWidthEq = 6.0,
+    latWidthPole = 9.0
+    ):
```

Comment:
@mark-petersen, I did this work in #518 so we're competing with one another on this.  I made the docstring have numpydoc formatting (allowing it to be automatically parsed and formatted the arguments in a more standard way -- one line per argument really isn't a standard practice.  In any case, we need to reconcile these differences before the next mega-merge.

Reply:
done.